### PR TITLE
DOC: Fix rst math for MixedLinear

### PR DESCRIPTION
**Pull request by saketkc**
I started preparing my proposal for GSoC, and hence.

### CODE REVIEW

File: docs/source/mixed_linear.rst [hunk 1]
```diff
@@ -30,14 +30,14 @@ hope to implement these features for the next release.
...
 
   import statsmodels.api as sm
   import statsmodels.formula.api as smf
 
-  data = sm.datasets.get_rdataset("dietox", "geepack").data
-  
-  md = smf.mixedlm("Weight ~ Time", data, groups=data["Pig"])
+  data = sm.datasets.get_rdataset(*dietox*, *geepack*).data
+
+  md = smf.mixedlm(*Weight ~ Time*, data, groups=data[*Pig*])
```
Comment (josef-pkt) [inline docs/source/mixed_linear.rst]: Why did you replace the quotes?   They are part of the code.

Comment (saketkc) [inline docs/source/mixed_linear.rst]: Sorry, did a `/g` replace for ". Reverting.

**Comment by saketkc** [comment]
Good to merge @josef-pkt?

**Comment by josef-pkt** [comment]
I think it looks fine, math in rst documentation is fine. I need to read through it again before merging.
(maybe add spelling to greek letters when they show up the first time, Phi ?)

@kshedden any comment

**Comment by kshedden** [comment]
Yes, it looks OK to me.

**Comment by josef-pkt** [comment]
Thanks kshedden

merging, thanks @saketkc

**Pull request merged**